packages/sentry/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hono/sentry",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
We don't have to modify the `version` field since the changeset will update it automatically.

Reply:
https://github.com/honojs/middleware/pull/620/commits/554a45f4dcafe84e83a9b9efb814c416835c136b